special-pages/pages/new-tab/public/locales/es/new-tab.json

```diff
@@ -333,12 +333,12 @@
     "note" : "Placeholder message indicating that no ads and trackers are detected"
   },
   "activity_no_adsAndTrackers_blocked" : {
-    "title" : "Sin anuncios + Intentos de rastreo bloqueados",
+    "title" : "Sin anuncios + intentos de rastreo bloqueados",
```

Comment:
Does changing a translated string manually here instead of going through the Smartling flow have any bad consequences?

Reply:
I’ll just run a new Smartling job anyway to be safe 🙂